clusters/build-clusters/build09/openshift-machine-api/machineset-worker-infra.yaml

```diff
@@ -0,0 +1,203 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build09-x99t2
+  name: build09-x99t2-worker-us-east-2a-infra
```

Comment:
```
oc --context build09 get machineset -n openshift-machine-api | grep infra
build09-x99t2-worker-us-east-2a-infra   1         1         1       1           92m
build09-x99t2-worker-us-east-2b-infra   1         1         1       1           92m
build09-x99t2-worker-us-east-2c-infra   1         1         1       1           92m
```
Are `2b` and `2c` leftover on the cluster, or were not brought to this PR by mistake?

Reply:
Intentional for redundancy